Uzoamaka126/My-Auto-Mart-Challenge
# [feature #165907312]Implementation of Admin Dashboard

#### What does this PR do?
Implement admin Dashboard interface

#### Description of Task to be completed?
- [x] Design and implementation of AutoMart Admin Dashboard interface

#### How should this be manually tested?
* Clone https://github.com/Uzoamaka126/My-Auto-Mart-Challenge
* Visit `/admin.html` on your browser
* You should see the image below

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#165907312

#### Screenshots (if appropriate)


#### Questions:
None
